daly-bms-uart.h

```diff
@@ -77,6 +75,15 @@ class Daly_BMS_UART
         // data from 0x97
         bool cellBalanceState[48];
         bool cellBalanceActive;
+        == == == =
```

Comment:
Something ist wrong here

Reply:
That's my bad

Reply:
😅 OK i was thinking was my Foul of May Keyboard, Sometimes it write curios letter